sdk/monitor/arminsights/ci.yml

```diff
@@ -2,14 +2,14 @@
 trigger:
   paths:
     include:
-    - sdk/monitor/armmonitor
+    - sdk/monitor/arminsights
 
```

Comment:
Can you confirm with the service team what the name should be?  I recall a few years ago they underwent some rebranding, I don't recall if we had actually fixed that in track 1 due to the breaking change.